roles/federation-demo/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# defaults file for federation-demo
+GITHUB_ACCESS_TOKEN: "token"
+KEY_TITLE: "evironment-vm"
+KEY_PATH: "~/.ssh/id_rsa.git"
+deployment_env : "test"
+GIT_REPO: "git@github.com:rciam/rciam-federation-registry.git"
+GIT_BRANCH: "devel"
+CLONE_DEST: "/var/www/rciam-federation-registry"
+KNOWN_HOSTS_PATH: "~/.ssh/known_hosts"
+GIT_EXECUTABLE: "/usr/bin/git"
+db_name: service_registry_db
+db_username: postgres
+db_password: postgres
+db_host: localhost
+ams_agent_key : "authrization_key"
+ams_token : "ams_token"
+ams_host: "https://msg-devel.argo.grnet.gr/v1"
+ams_project: "rciam-service-registry"
+ams_sub: "express_push"
+ams_topic: "deployment_response"
+db_script_path: /home/debian/setup_db.sql
```

Comment:
If you want, it suits to your use cases and you agree that it is more beautiful, I would suggest you to "group" the variables in lists, for example:
```
git :
  REPO: "git@github.com:rciam/rciam-federation-registry.git"
  BRANCH: "devel"
  GITHUB_ACCESS_TOKEN: "token"
  CLONE_DEST: "/var/www/rciam-federation-registry"
  GIT_EXECUTABLE: "/usr/bin/git"

db:
  name: service_registry_db
  username: postgres
  password: postgres
  host: localhost
  script_path: /home/debian/setup_db.sql

ams:
  agent_key : "authrization_key"
  token : "ams_token"
  host: "https://msg-devel.argo.grnet.gr/v1"
  project: "rciam-service-registry"
  sub: "express_push"
  topic: "deployment_response"

KEY_TITLE: "evironment-vm"
KEY_PATH: "~/.ssh/id_rsa.git"
deployment_env : "test"

KNOWN_HOSTS_PATH: "~/.ssh/known_hosts"
```